src/components/Nodes/wrapNode.tsx

```diff
@@ -146,36 +157,13 @@ export default (NodeComponent: ComponentType<NodeProps>) => {
 
     const onDragStop = useCallback(
       (event: UseDragEvent) => {
-        // onDragStop also gets called when user just clicks on a node.
-        // Because of that we set dragging to true inside the onDrag handler and handle the click here
-        let node;
-
-        if (onClick || onNodeDragStop) {
-          node = store.getState().nodeInternals.get(id)!;
-        }
-
-        if (!dragging) {
-          if (isSelectable && !selectNodesOnDrag && !selected) {
-            addSelectedNodes([id]);
-          }
-
-          if (onClick && node) {
-            onClick(event.sourceEvent as MouseEvent, { ...node });
-          }
-
-          return;
-        }
-
-        updateNodePosition({
-          id,
-          dragging: false,
-        });
-
-        if (onNodeDragStop && node) {
-          onNodeDragStop(event.sourceEvent as MouseEvent, { ...node, dragging: false });
+        setDragging(false);
```

Comment:
by removing the code from the original onDragStop handler you broke the `onClick` handler for draggable nodes. Could you add lines 151 - 167 again? thanks!